tests/content/src/main/content/jcr_root/content/kyanite-visual-tests/pages/blog-article-author-bio/.content.xml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:ws="http://ds.pl/websight" xmlns:sling="http://sling.apache.org/jcr/sling/1.0"
+    jcr:primaryType="ws:Page">
+    <jcr:content
+        jcr:lastModified="{Date}2023-06-01T12:21:43.245Z"
+        jcr:lastModifiedBy="wsadmin"
+        jcr:primaryType="ws:PageContent"
+        jcr:title="Blog article author bio"
+        sling:resourceType="kyanite/blogs/components/blogarticlepage"
+        ws:lastModified="{Date}2023-06-02T09:13:21.799Z"
+        ws:lastModifiedBy="wsadmin"
+        ws:template="/libs/kyanite/blogs/templates/blogarticlepage"
+        authorDescription="Goal oriented and not afraid to step out of his comfort zone. He specializes in AEM based websites development and maintenance, including AEM components. Paweł is interested in chess, sports and time with his family."
+        authorName="Anna Szemiot"
```

Comment:
And, let's rename author to John Doe in all the pages?

Reply:
okey, done